mobile/src/framework/notifications/types/show-message-arguments.ts

```diff
@@ -1,6 +1,6 @@
 type ShowMessageArguments = {
-    title: string;
-    text?: string;
+    text1: string;
```

Comment:
why did you decide to change it?

Reply:
Toast did not work, so I changed the names.

Reply:
So let's try to investigate why it doesn't work

Reply:
If someone pooped in the package, we will not pull all this into our project)

Reply:
we don't use such naming

Reply:
I return to the previous naming